q2_demux/_summarize/_visualizer.py

```diff
@@ -108,6 +123,7 @@ def summarize(output_dir: str, data: _PlotQualView, n: int=10000) -> None:
         count = 0
         for seq in _read_fastq_seqs(file):
             count += 1
+
```

Comment:
Oh I'm really sorry I was looking at the wrong chunk of code.  I have no idea why but I thought you were talking about line 214 of `_visualizer.py`.  My mistake!  Yes you're right that's unnecessary whitespace, I'll fix that.  Thanks!